README.md

```diff
@@ -27,6 +27,14 @@ composer install
 vendor/bin/phpunit
 ```
 
+or you may use;
+
+```bash
+composer install
+composer check
```

Comment:
Just confirming `composer check` and `composer test` both are same right.

Reply:
yep